src/scaffold/index.js

```diff
@@ -48,10 +49,12 @@ module.exports = class Scaffold {
             `--node http://localhost:8020/ ` +
             `--ipfs http://localhost:5001 ` +
             this.subgraphName,
+          'test': 'graph test',
         },
         dependencies: {
           '@graphprotocol/graph-cli': GRAPH_CLI_VERSION,
           '@graphprotocol/graph-ts': `0.27.0`,
+          'matchstick-as': `0.5.0`
```

Comment:
Nice! \o/ 🎉

Reply:
Shouldn't `matchstic-as` and `@graphprotocol/graph-cli` be `dev-dependencies`?